Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds.py

```diff
@@ -0,0 +1,223 @@
+                        if SNX_CREATED_KEY in i.keys():
+                            timeData = i[SNX_CREATED_KEY]
+                            createdValue = datetime.strptime(timeData, '%Y-%m-%dT%H:%M:%S.%fZ').strftime(DEMISO_DATE_FORMAT)
+                        if SNX_MODIFIED_KEY in i.keys():
+                            timeData = i[SNX_MODIFIED_KEY]
+                            modifiedValue = datetime.strptime(timeData, '%Y-%m-%dT%H:%M:%S.%fZ').strftime(DEMISO_DATE_FORMAT)
+                        indicatorObj = {}
+                        if indicatorValue is not None:
+                            indicatorObj["value"] = indicatorValue
+                        else:
+                            continue
+                        if indicatorType is not None:
+                            indicatorObj["type"] = indicatorType
+                        else:
+                            continue
+                        indicatorObj["rawJson"] = i
+                        indicatorObj["fields"] = {
+                            "firstseenbysource": createdValue,
+                            "indicatoridentification": indicatorIdValue,
+                            "verdict": "Malicious",
+                            "tags": tagList,
+                            "modified": modifiedValue,
+                            "reportedby": reportByValue
+                        }
+                        indicatorList.append(indicatorObj)
+
+                    except Exception as e:
+                        demisto.error(e)
+
+    except Exception as e:
+        demisto.error(e)
+    return indicatorList
+
+
+def getIndicatorsCommand(client: Client, args: Dict[str, str]) -> CommandResults:
+    count = arg_to_number(args.get('limit')) or 2
+    feedJson = client.get_feeds(None)
+    if feedJson is not False and feedJson != {}:
+        if SNX_OBJECTS_KEY in feedJson.keys():
+            indicatorList = parseIndicators(feedJson)
+            finalIndicatorList = indicatorList[:count]
+            readableOutput = tableToMarkdown('SecneurX Threat Feeds Indicators: ',
+                                             t=finalIndicatorList, headers=['type', 'value', 'fields'])
+            command_result = CommandResults(
+                outputs_prefix='',
+                outputs_key_field='',
+                outputs={},
+                readable_output=readableOutput,
+                raw_response=finalIndicatorList
+            )
+            return command_result
+        else:
+            raise DemistoException('Indicators not found in feed')
+    else:
+        raise DemistoException('Cannot fetch the feed')
+
+
+def fetchThreatFeeds(client: Client, paramsValue: Optional[str] = None) -> Any:
+    indicators = []
+    try:
+        feedList = client.get_feeds(paramsValue)
+        if feedList is not False:
+            indicators = parseIndicators(feedList)
+
+    except Exception as e:
+        demisto.error(e)
+    return indicators
```

Comment:
Please raise an exception in case the get_feeds or parsing the indicators was failed 
and don't catch it here - it will handle in the main 
think about a case that there is error in such flow
how the user will notify about this ?
currently it will only appear in the log

Reply:
Agreed.